pro/src/new_components/VenueForm/Informations/validationSchema.ts

```diff
@@ -1,7 +1,14 @@
 import * as yup from 'yup'
 
 const validationSchema = {
-  publicName: yup.string().required('Veuillez renseigner un nom public'),
+  mail: yup
+    .string()
+    .required('Veuillez renseigner une adresse e-mail')
+    .email(
+      'L’e-mail renseigné n’est pas valide. Exemple : votrenom@votremail.com'
```

Comment:
```suggestion
    .required('Veuillez renseigner un email')
    .email(
      'Veuillez renseigner un email valide'
```
désolé j'ai eu la version définitive des ux hier :)
maintenant c'est ça partout